policy/daemon.go

```diff
@@ -317,11 +316,16 @@ func (p *policyd) fetchPolicy(ctx context.Context, domain string) (*SignedPolicy
 		glg.Warn(errors.Wrap(err, "error body.close"))
 	}
 
-	// set eTag cache
-	eTag := res.Header.Get("ETag")
-	if eTag != "" {
-		glg.Debugf("Setting ETag %v for domain %s", eTag, domain)
-		p.etagCache.SetWithExpire(domain, &etagCache{eTag, sp}, p.etagExpTime)
+	// set etag cache
+	etag := res.Header.Get("ETag")
+	if etag != "" {
+		etagValidDur := sp.SignedPolicyData.Expires.Time.Sub(fastime.Now()) - p.expireMargin
+		glg.Debugf("Set domain %s with etag %v, duration: %s", domain, etag, etagValidDur)
```

Comment:
```suggestion
		glg.Debugf("set domain %s with etag %v, duration: %s", domain, etag, etagValidDur)
```

Reply:
some log messages start with 'A', some with 'a'.
I cannot find the pattern...